CHANGELOG.md

```diff
@@ -19,3 +19,4 @@ We refer to [GitHub issues](https://github.com/eclipse/winery/issues) by using `
 
 ### Fixed
 * Bounary definitions can be browsed for exported operations again
+* 
```

Comment:
Is here text missing? Here, only fixes from the last official release are given. Since there is no release, there should be no entries here 😇